Sources/tart/VMStorageOCI.swift

```diff
@@ -170,7 +170,7 @@ class VMStorageOCI: PrunableStorage {
 
     if !exists(digestName) {
       let transaction = SentrySDK.startTransaction(name: name.description, operation: "pull", bindToScope: true)
-      let tmpVMDir = try VMDirectory.temporary()
+      let tmpVMDir = try VMDirectory.temporaryDeterministic(fileName: name.namespace)
```

Comment:
Seems either `digestName` or even full `name.description`. `name.description` contains slashes and might not be file safe so I think in this case you can rename parameter of `temporaryDeterministic` from `fileName` to something like `key` and hash the `key` using [CryptoKit](https://developer.apple.com/documentation/cryptokit/). Some insecure hashing like md5 will work just fine.